code/modules/admin/admin_verbs.dm

```diff
@@ -1186,3 +1187,16 @@ GLOBAL_PROTECT(admin_verbs_poll)
 		QDEL_NULL(segment.ai_controller)
 		segment.AddComponent(/datum/component/mob_chain, front = previous)
 		previous = segment
+
+/client/proc/spawn_as_mmi(mob/living/carbon/human/target in GLOB.human_list)
+	set category = "Debug"
+	set name = "Turn target into MMI"
+	set desc = "Turns something into an MMI, must be used on humans"
```

Comment:
Add a check so it isn't used on non-humans then.

Reply:
figured the target in human list would be enough, added a ishuman check